.pre-commit-config.yaml

```diff
@@ -37,6 +37,6 @@
     files: ".*/static2/.*graphql$"
   - id: flake8
     name: flake8
-    entry: bash -c "PATH=./venv/bin/:$PATH flake8 `git diff --name-only --diff-filter=d --cached -- '*.py'|grep assembl/graphql`"
+    entry: bash -c "PATH=./venv/bin/:$PATH flake8 `git diff --name-only --diff-filter=d --cached -- '*.py'|grep assembl/`"
     language: system
-    files: "assembl/graphql/.*.py"
+    files: "assembl/"
```

Comment:
This is not a valid expression for `files`. You should remove files here and add
use `types: [python]` instead.

Reply:
done